README.md

```diff
@@ -1,26 +1,107 @@
 
-Checkout `source`, then run `ember s` to serve and develop locally.
+## To Add/Update a Project
 
-**Adding your project**
+Add the project and details as a JSON object in the appropriate project group's `"projects"` array in `src/data/projects.json`.
 
-Add your project + details to the `groups` property of [the application controller](https://github.com/tedconf/tedconf.github.io/blob/source/app/pods/application/controller.js).
+Then `yarn build` and deploy to `master` branch.
 
-**Deploying**
+**TODO: Deploy docs? The Ember-based deploy stuff seemed extra complicated, but nothing has been done to script the delivery of the build output to the `master` branch root.**
 
-When you're ready to deploy your changes, make sure you're on the `source` branch, then run
+## Project Component Structure
 
-```sh
-$ ember github-pages:commit --branch master --message 'Deploying to master'
-# The ember app will build, then you'll see this message:
-Done. All that's left is to git push the master branch.
-$ git push origin master
+The rendering tree looks something like this. A few notes on notation:
+
+* Data imports are shown in parens next to the component that imports the data file.
+* A `+` next to the component means many of those are potentially rendered depending on `props`.
+
+```{text}
+index.js
+'- src/components/App (<- src/data/projects.json)
+    |- src/components/AppHeader
+    |- src/components/ProjectGroups
+    |   '- src/components/ProjectGroup +
+    |       '- src/components/ProjectCard +
+    '- src/components/Footer (<- src/data/footer-links.json)
 ```
 
-and in not too long, http://tedconf.github.io/ should be updated with your changes!
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start` or `yarn start`
+
+Runs the app in the development mode.<br>
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br>
+You will also see any lint errors in the console.
+
+### `npm test` or `yarn test`
+
+Launches the test runner in the interactive watch mode.<br>
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build` or `yarn build`
+
+Builds the app for production to the `build` folder.<br>
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.<br>
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run deploy` or `yarn run deploy`
+
+**Note: this has not been tested yet!**
+
+Based on this: https://facebook.github.io/create-react-app/docs/deployment#github-pages-https-pagesgithubcom
+
+This uses the `gh-pages` package to build the app and push the built code to `master` branch.
+
+### `npm run eject`
```

Comment:
__*Preference*__

Would clean up the readme a bit, pull out some of the generic CRA stuff and make it more specific to this project.

Reply:
I did some cleanup already. I left what was there as I didn't know if it would be needed later. Feel free to revise as needed.